extensions.html

```diff
@@ -334,6 +334,117 @@ <h2><dfn>GamepadPose</dfn> Interface</h2>
         </dd>
       </dl>
     </section>
+    
+    <section>
+      <h2><dfn>GamepadLightIndicatorType</dfn> Enum</h2>
+      <p>
+        This enum defines the type of light indicators supported by gamepads.
+      </p>
+      
+      <pre class="idl">
+        enum GamepadLightIndicatorType {
+          "on-off",
+          "rgb"
+        };
+      </pre>
+      
+      <dl data-dfn-for="GamepadLightIndicatorType">
+        <dt>"<dfn>on-off</dfn>"</dt>
+        <dd>
+          Light indicator that supports a single color.
+        </dd>
+        
+        <dt>"<dfn>rgb</dfn>"</dt>
+        <dd>
+          Light indicator that supports multiple colors.
+        </dd>
+      </dl>
+    </section>
+    
+    <section>
+      <h2><dfn>GamepadLightColor</dfn></h2>
+      <p>
+        This represents the color of a light indicator. The default value should
+        be set to 0 for red, green, and blue.
+      </p>
+    
+      <pre class="idl">
+        dictionary GamepadLightColor {
+          octet red = 0;
+          octet green = 0;
+          octet blue = 0;
+       };
+      </pre>
+      
+      <dl data-dfn-for="GamepadLightColor">
+        <dt><dfn>red</dt> member</dfn>
+        <dd>
+          Red component of the light color, or non-zero value for an
+          <code>on-off</code> light indicator that indicates ON.
+        </dd>
+        
+        <dt><dfn>green</dt> member</dfn>
+        <dd>
+          Blue component of the light color, or non-zero value for an
+          <code>on-off</code> light indicator that indicates ON.
+        </dd>
+        
+        <dt><dfn>blue</dt> member</dfn>
+        <dd>
+          Green component of the light color, or non-zero value for an
+          <code>on-off</code> light indicator that indicates ON.
+        </dd>
+      </dl>
+    </section>
+    
+    <section>
+      <h2><dfn>GamepadLightIndicator</dfn></h2>
+      <p>
+        This interface defines a light indicator.
+      </p>
+      
+      <pre class="idl">
+        [Exposed=Window, SecureContext]
+        interface GamepadLightIndicator {
+          readonly attribute GamepadLightIndicatorType type;
+        
+          Promise&lt;long&gt; setColor(GamepadLightColor color);
```

Comment:
Given the discussion below, the promise should probably just resolve with `undefined`. 

```suggestion
          Promise&lt;undefined&gt; setColor(GamepadLightColor color);
```